.travis.yml

```diff
@@ -0,0 +1,7 @@
+language: ruby
+rvm:
+  - "1.9.3-p194"
```

Comment:
Fixed.  Also caused a 'gem install bundler' to be put in the build script - FYI.